Configure

```diff
@@ -917,6 +917,12 @@ while (@argvcopy)
                         push @{$useradd{CXXFLAGS}}, $_;
                         }
                 }
+        elsif (m|^/|)
+                {
+                $_ =~ s/%([0-9a-f]{1,2})/chr(hex($1))/gei;
```

Comment:
This mysterious substitution turned out to be a little (undocumented?) feature introduced in commit f6666ad19d24 to support passing compiler flags with arguments separated by spaces. (`-foo%20bar` get's translated to `-foo bar`).

Supporting abitrary ASCII codes seems a bit hacky and is overengineered for just adding spaces. Instead of moving this hack to the Windows world, I would rather suggest to add an officially documented substitution for adding space characters, which is more easy to remember: Something like `-foo[]bar` -> `-foo bar`, what do you think @levitte?

The same substitution would be added to the UNIX-style options, leaving the hack in place for backward compatibility.

Reply:
Personally, I find the %xx hack horrible, but apparently, someone wanted it, and I can see the benefit compared to using quotes and having to figure out how quoted within quotes are interpreted.
On the plus side, the %xx syntax *is already known* from URI syntax.  Reinventing a new ad-hoc syntax doesn't make it better.  I would rather keep the %xx syntax, if we must have something like that at all.

Reply:
>  benefit compared to using quotes and having to figure out how quoted within quotes

Ah, interesting argument. After a night of sleep I was just starting to wonder why people not just simply quote the argument.  

> I would rather keep the %xx syntax, if we must have something like that at all.

Ok, agreed. I'll just add a short explaining comment above the substitution (here and above line 915). Something

```
# treat %xx  as an ascii code (e.g. replace %20 by a space character,  "foo%20bar" -> "foo bar") 
```